invite/settings.py

```diff
@@ -36,4 +36,4 @@ def get_service_name():
     return getattr(
         settings,
         'INVITE_SERVICE_NAME',
-        Site.objects.get_current().name)
+        Site.objects.get_current(request=request).name)
```

Comment:
Do we need to have `sites` as an installed app in projects where we aren't setting a SITE_ID? Also, I guess the proper function to have suggested would more properly have been `django.contrib.sites.shortcuts.get_current_site`.